patsy.go

```diff
@@ -52,7 +58,35 @@ func Dir(env vos.Env, packagePath string) (string, error) {
 	}
 
 	return "", errors.Errorf("Dir not found for %s", packagePath)
+}
+
+// Dirs returns the filesystem path for all packages under the directory corresponding to the go
+// package path provided.
+func Dirs(env vos.Env, packagePath string) (map[string]string, error) {
+	exe := exec.Command("go", "list", "-f", "{{.ImportPath}}:{{.Dir}}", packagePath)
+	exe.Env = env.Environ()
+	out, err := exe.CombinedOutput()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	lines := strings.Split(string(out), "\n")
```

Comment:
afaik go commands output `\n` on windows as well and this is safe, no need to use `\r\n`